src/main/java/core/basesyntax/JavaDateTimeApi.java

```diff
@@ -31,39 +50,47 @@ public String todayDate(DateTimePart datePart) {
      *                   - 3-й элемент массива - день (число);
      */
     public Optional<LocalDate> getDate(Integer[] dateParams) {
-        return Optional.empty();
+        if (dateParams.length < 1) {
+            return Optional.empty();
```

Comment:
do we need this checking

Reply:
We need this check to don't get OutOfBound